webclient/dokument/forms.py

```diff
@@ -247,6 +277,9 @@ def __init__(
         self.fields["rada"].initial = rada
         self.fields["let"].initial = let
         self.fields["dokument_osoba"].initial = dok_osoby
+        self.fields["format"].choices = [("", "")] + heslar_list(
+            HESLAR_DOKUMENT_FORMAT, {"id__in": MODEL_3D_DOKUMENT_FORMATS}, True
```

Comment:
@jnihnat Nemělo by tu být negace?

> Při editaci dokumentu (např. https://amcr-test.aiscr.cz/dokument/edit/C-TX-193001369) omezit výběr v poli Formát plánu/foto na hodnoty, které **neodpovídají** MODEL_3D_DOKUMENT_FORMATS.

Reply:
To se dela v te metode diky tomu True na konci.

Reply:
Aha, napadlo mě to, ale nevěděl jsem, jak to číst. Díky.